### PR TITLE
Email create

### PR DESCRIPTION
**Pull request by malamadhu**
Sending Mail Changes

### CODE REVIEW

File: BookingSystem.BAL/Interface/IHotelService.cs [hunk 1]
```diff
@@ -11,6 +11,8 @@ public interface IHotelService
     {
         Task<List<HotelVM>> GetHotelDetailsAsync();
```
Comment (bhairab15) [inline BookingSystem.BAL/Interface/IHotelService.cs:13]: Some blank lines added

File: BookingSystem.BAL/Service/MailService.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            builder.HtmlBody = mailRequest.Body;
+            email.Body = builder.ToMessageBody();
+            using var smtp = new SmtpClient();
+            smtp.Connect(_mailSettings.Host, _mailSettings.Port, SecureSocketOptions.StartTls);
+            smtp.Authenticate(_mailSettings.Mail, _mailSettings.Password);
+            await smtp.SendAsync(email);
+            smtp.Disconnect(true);
+        }
+    }
+    public class MailRequest
```
Comment (bhairab15) [inline BookingSystem.BAL/Service/MailService.cs:54]: Please move these classes to different files

Comment (manic1990) [inline BookingSystem.BAL/Service/MailService.cs:54]: Should create all  model class inside model library

File: BookingSystem.BAL/Service/MailService.cs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+namespace BookingSystem.BAL.Service
+{
+    public class MailService : IMailService
+    {
+        private readonly MailSettings _mailSettings;
+        public MailService(IOptions<MailSettings> mailSettings)
+        {
+            _mailSettings = mailSettings.Value;
+        }
+        public async Task SendEmailAsync(MailRequest mailRequest)
```
Comment (manic1990) [inline BookingSystem.BAL/Service/MailService.cs:22]: Use return type bool.

File: BookingSystemAPI/Controllers/EMailController.cs [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class EMailController : ControllerBase
+    {
+        private readonly IMailService mailService;
+        private readonly ILogger _logger;
+        private const string started = "method started";
+        private const string completed = "method completed";
+        public EMailController(IMailService mailService, ILoggerFactory logFactory)
```
Comment (manic1990) [inline BookingSystemAPI/Controllers/EMailController.cs:23]: Create async mrthod.

**Review by malamadhu — COMMENTED** [review]
resolved comments added

**Review by bhairab15 — APPROVED** [review]

**Pull request merged**